src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisShardSyncer.java

```diff
@@ -62,13 +63,14 @@ synchronized void bootstrapShardLeases(IKinesisProxy kinesisProxy, ILeaseManager
             boolean ignoreUnexpectedChildShards)
             throws DependencyException, InvalidStateException, ProvisionedThroughputException,
             KinesisClientLibIOException {
-        syncShardLeases(kinesisProxy, leaseManager, initialPositionInStream, cleanupLeasesOfCompletedShards,
+        syncShardLeases(null, kinesisProxy, leaseManager, initialPositionInStream, cleanupLeasesOfCompletedShards,
                 ignoreUnexpectedChildShards);
     }
 
     /**
      * Check and create leases for any new shards (e.g. following a reshard operation).
      *
+     * @param shards
```

Comment:
It is recommended to add new param at the end. Also add a comment on how these shards are used.

Reply:
Ok, will move to the end and add comment